charts/anchor-platform/sep-service/templates/configmap.yaml

```diff
@@ -1,7 +1,19 @@
+{{- if .Values.assets }}
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: {{ .Values.fullName }}-assets
```

Comment:
You don't seem to be consuming this ConfigMap anywhere

Reply:
It's a little non-obvious, but the volume mounts are also configured by name in values.yaml.